app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const {errorHandler} = require('./db/error-handler')
+const app = express();
+const {getTopics} = require('./controllers/topics-controllers')
+app.use(express.json());
```

Comment:
I know the temptation is to put this in at the start, but you don't need it yet. In future only add code that is relevant to the ticket 👍